frontend/src/pages/Mesh/Mesh.tsx

```diff
@@ -188,7 +190,13 @@ const TopologyContent: React.FC<{
       requestFit = true;
       controller.getGraph().reset();
       controller.getGraph().layout();
+
+      // Fit padding after resize
+      setTimeout(() => {
+        controller.getGraph().fit(FIT_PADDING);
+      }, 0);
     }
```

Comment:
As you know, I spent a while trying to solve this issue and I had considered a timeout but thought that really this was an issue in PFT, and that it needed to be more responsive to resize.  I don't really know why adding this to the end of the queue works, but I'm glad it does!  We may be able to remove the "onResize" infra I added while trying to solve the problem, I don't think it's necessary.  I can do that in my current PR, as well as apply this to GraphPF.